README.md

```diff
@@ -18,9 +18,9 @@ https://VMware.zoom.us/j/3697467292
 ## Running in docker
 
 ```
-docker pull badboyqiao/edgex-ui-go:0.1
+docker pull badboyqiao/edgex-ui-go:0.1.1
```

Comment:
as an aside, do we not have this docker image hooked up with the official edgexfoundry account? I think we should try to do that for Edinburgh

Reply:
ok，i will update it